cmd/access/access_node_builder.go

```diff
@@ -53,20 +53,32 @@ func FlowAccessNode() *FlowAccessNodeBuilder {
 		FlowNodeBuilder: cmd.FlowNode(flow.RoleAccess.String()),
 	}
 }
-func (anb *FlowAccessNodeBuilder) IsStaked() bool {
-	return anb.staked
+func (builder *FlowAccessNodeBuilder) IsStaked() bool {
+	return builder.staked
 }
 
-func (anb *FlowAccessNodeBuilder) parseFlags() {
+// supportUnstakedNodes returns True if this a staked Access Node which also participates in the unstaked network,
+// False otherwise
+func (builder *FlowAccessNodeBuilder) supportUnstakedNodes() bool {
```

Comment:
What I meant was to add it to the `AccessNodeBuilder` interface, not only the `FlowAccessNodeBuilder` implementation. It will also need to be public. See [this](https://github.com/onflow/flow-go/commit/59ae3f825929ceef58390657367ece1a0ef71aa2#r53749799) for example.

We would rename the method as `ParticipatesInUnstakedNetwork`, and it would always return true for unstaked nodes. 

Of course, instead of doing this I could just do a type assertion [here](https://github.com/onflow/flow-go/commit/59ae3f825929ceef58390657367ece1a0ef71aa2#r53750040), but I think this way is a little bit cleaner.

Reply:
oh shoot! I forgot to push the commit with that change. Yes, 100% agree that method should be on the interface.